obal/data/playbooks/library/rpmspec_changelog_check.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+from ansible.module_utils.basic import AnsibleModule
+
+import os
+import subprocess
+import glob
+
+def run_module():
+    module_args = dict(
+        directory=dict(type='str', required=True),
+    )
+
+    result = dict(
+        changed=False,
+        changelog_version_release='',
+        specfile_version_release=''
+    )
+
+    module = AnsibleModule(
+        argument_spec=module_args,
+        supports_check_mode=True
+    )
+
+    if module.check_mode:
+        return result
+
+    specfile = glob.glob(os.path.join(module.params['directory'], "*.spec"))[0]
```

Comment:
You could be very safe and do a `try: except IndexError: module.fail_json(msg='Could not find spec file', **result)` here